Reason: Skip tailwindcss integration test which fails on arches besides x86_64
and aarch64, limited by npm module watcher
Upstream: No

--- a/resources/resource_transformers/cssjs/tailwindcss_integration_test.go
+++ b/resources/resource_transformers/cssjs/tailwindcss_integration_test.go
@@ -14,17 +14,17 @@
 package cssjs_test
 
 import (
+  "os"
 	"testing"
 
 	"github.com/bep/logg"
-	"github.com/gohugoio/hugo/htesting"
 	"github.com/gohugoio/hugo/hugolib"
 )
 
 func TestTailwindV4Basic(t *testing.T) {
-	if !htesting.IsCI() {
-		t.Skip("Skip long running test when running locally")
-	}
+  if os.Getenv("GOARCH") != "amd64" && os.Getenv("GOARCH") != "arm64" {
+    t.Skip("Skip tailwindcss integration test")
+  }
 
 	files := `
 -- hugo.toml --
